packages/destination-actions/src/destinations/amazon-eventbridge/index.ts

```diff
@@ -0,0 +1,85 @@
+
+import send from './send'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Amazon Eventbridge',
+  slug: 'actions-amazon-eventbridge',
+  mode: 'cloud',
+  description:
+    'Amazon EventBridge is a serverless event bus service that makes it easy to connect your applications with data from a variety of sources.',
+  authentication: {
+    scheme: 'custom',
+    fields: {
+      awsAccountId: {
+        type: 'string',
+        label: 'AWS Account ID',
+        description: `The AWS Account ID that the event bus belongs to. 
+                      This is used to generate the ARN for the event bus.`,
+        required: true
+      },
+      awsRegion: {
+        type: 'string',
+        label: 'AWS Region',
+        description: 'The AWS region that the event bus belongs to.',
+        disabledInputMethods: ['enrichment', 'function', 'variable'],
+        required: true,
+        choices: [
+          { label: 'us-east-1', value: 'us-east-1' },
+          { label: 'us-east-2', value: 'us-east-2' },
+          { label: 'us-west-1', value: 'us-west-1' },
+          { label: 'us-west-2', value: 'us-west-2' },
+          { label: 'eu-west-1', value: 'eu-west-1' },
+          { label: 'eu-west-2', value: 'eu-west-2' },
+          { label: 'eu-west-3', value: 'eu-west-3' },
+          { label: 'ap-southeast-1', value: 'ap-southeast-1' },
+          { label: 'ap-southeast-2', value: 'ap-southeast-2' },
+          { label: 'sa-east-1', value: 'sa-east-1' },
+          { label: 'ap-northeast-1', value: 'ap-northeast-1' },
+          { label: 'ap-northeast-2', value: 'ap-northeast-2' },
+          { label: 'ap-south-1', value: 'ap-south-1' },
+          { label: 'ca-central-1', value: 'ca-central-1' },
+          { label: 'eu-central-1', value: 'eu-central-1' }
+        ]
+      },
+      partnerEventSourceName: {
+        type: 'string',
+        label: 'Partner Event Source Name',
+        description: 'The name of the partner event source to use for the event bus.',
+        required: true,
+        default: 'segment.com',
+        choices: [
+          { label: 'segment.com', value: 'aws.partner/segment.com' },
+          { label: 'segment.com.test', value: 'aws.partner/segment.com.test' }
+        ]
+      },
+      createPartnerEventSource: {
+        type: 'boolean',
+        label: 'Create Partner Event Source',
+        description: `NOTE: This will create the partner event source if it does not already exist.
+                      Use this option if you want to create the partner event source.
+                      projectId or context.protocols.sourceId will be used as the sourceId to 
+                      create the partner event source.
+                      Use with caution.`,
+        default: false
+      }
+    },
+    testAuthentication: async (_request, { settings }) => {
+      if (!settings.awsAccountId || settings.awsAccountId === '' || !settings.awsRegion || settings.awsRegion === '') {
```

Comment:
Empty strings evaluate to`false` in Javascript, so this should suffice:

```suggestion
      if (!settings.awsAccountId || !settings.awsRegion {
```